src/util/records.py

```diff
@@ -1,8 +1,8 @@
 # -*- coding: utf-8 -*-
+# ruff: noqa: F403, F405  # ruff is unable to detect * imports
```

Comment:
This is unfortunate. This avoids catching real undefined local variable errors.
```suggestion
```

Reply:
True. AFAIK Ruff still doesn't support multi-file analysis :disappointed:, hopefully cases like these will be fixed in the future.